### PR TITLE
Auto updated assets updated for go models and clients

### PR DESCRIPTION
**Pull request by yograjshisode**

### CODE REVIEW

File: java/src/com/vmware/avi/sdk/AviAuthorizationInterceptor.java [hunk 1]
```diff
@@ -82,11 +81,7 @@ public ClientHttpResponse intercept(HttpRequest request, byte[] body, ClientHttp
 					}
 				}
 			}
-		} catch (IOException e) {
-			e.printStackTrace();
-			throw e;
-		} catch (InterruptedException e) {
-			// TODO Auto-generated catch block
+		} catch (Throwable e) {
```
Comment (chaitanyaavi) [inline java/src/com/vmware/avi/sdk/AviAuthorizationInterceptor.java:84]: This change is not expected @tusharlimbhore Can you please check this if it needs to update template

File: java/src/com/vmware/avi/sdk/AviCredentials.java [hunk 2]
```diff
@@ -92,7 +92,7 @@ public AviCredentials(String controller, String username, String password, Strin
     private String version;
     private String tenantUuid;
     private Integer port = 443;
-    private Integer timeout = 60; // 1 min default
+    private Integer timeout;
```
Comment (chaitanyaavi) [inline java/src/com/vmware/avi/sdk/AviCredentials.java:95]: Not Expected
@yograjshisode did you take the change from public repo to avi-dev?

Comment (yograjshisode) [reply java/src/com/vmware/avi/sdk/AviCredentials.java:95]: I'll sync avi-dev with public sdk and will create PR again. Thanks CD for pointing this one

**Comment by yograjshisode** [comment]
avi-dev sync is needed

**Pull request closed without merging**